### PR TITLE
style(deployments): Add tooltip to checkmark in deployment card

### PR DESCRIPTION
**Pull request by c-koehler**
fixes https://github.com/openshiftio/openshift.io/issues/1452

Note that this depends on https://github.com/fabric8-ui/fabric8-ui/pull/2252

### CODE REVIEW

**Comment by andrewazores** [comment]
I think it'd be safe enough to rebase this commit onto the master branch rather than my deployments-tab. It's a very small commit - a one-liner in one file.

File: src/app/space/create/deployments/components/deployment-card/deployment-card.component.html [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="card-pf" (click)="collapsed = !collapsed">
   <div class="card-pf-body">
-    <span class="pficon pficon-ok"></span>
+    <span class="pficon pficon-ok" data-toggle="tooltip" data-placement="top" title="Everything looks okay"></span>
```
Comment (andrewazores) [inline src/app/space/create/deployments/components/deployment-card/deployment-card.component.html]: When I apply this locally, the tooltip is not placed at the "top" and doesn't look like the tooltip as shown in the design mockup.

The patternfly docs point to the bootstrap docs: https://getbootstrap.com/docs/3.3/javascript/#tooltips

which says this: `For performance reasons, the Tooltip and Popover data-apis are opt-in, meaning you must initialize them yourself.`

I suspect that the tooltip data-api not being initialized is why I'm seeing it presented visually differently than the patternfly standard tooltip.

**Comment by c-koehler** [comment]
@andrewazores I ended up playing around with it and getting it to work with master (without including the 3 commits above). Do you still want me to do this? I believe you said to do this but i want to make sure before before I toast the 3 commits here. I've resolved all the issues for making it purely based on master so it should be a quick thing to do.

**Comment by andrewazores** [comment]
@c-koehler yes - "please rebase this onto master". Those three commits of mine from PR2252 should not be including in the history of this PR, otherwise accepting this PR means that my commits will also be merged into master, and that is not the point of this PR.

**Comment by c-koehler** [comment]
Updated/squashed and should be good to go!

**Comment by andrewazores** [comment]
Perfect, LGTM. Thanks.

(tiny nit - the A in "Add tooltip" for the commit message should not be capitalized)

**Comment by joshuawilson** [comment]
@c-koehler please fix the requested change and rebase.

**Comment by fabric8cd** [comment]
@c-koehler fabric8/fabric8-ui:SNAPSHOT-PR-2259-2 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2259-fabric8-ui.badger.fabric8.io

**Comment by fabric8cd** [comment]
@c-koehler fabric8/fabric8-ui:SNAPSHOT-PR-2259-4 fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2259-fabric8-ui.badger.fabric8.io

**Pull request merged**